lib/ansible/template/__init__.py

```diff
@@ -612,6 +631,17 @@ def _get_filters(self):
         for fp in self._filter_loader.all():
             self._filters.update(fp.filters())
 
+        if USE_JINJA2_NATIVE:
+            for string_filter in C.STRING_TYPE_FILTERS:
+                try:
+                    orig_filter = self._filters[string_filter]
+                except KeyError:
+                    try:
+                        orig_filter = FILTERS[string_filter]
+                    except KeyError:
+                        continue
+                self._filters[string_filter] = _wrap_native_text(orig_filter)
```

Comment:
Should we also do this in `JinjaPluginIntercept.__getitem__`?  Since this comes from a config, in order to work with something from a collection it would have to go there too I think.

Reply:
:+1: Done.